### PR TITLE
Moving Aptos CLI README docs to this docs site.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2081)

### CODE REVIEW

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Install Aptos CLI"
+id: "install-aptos-cli"
+---
+
+# Install Aptos CLI
+
+The `aptos` tool is a command line interface (CLI) for debugging, development, and node operation.
```
Comment (davidiw) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:8]: node operations?

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+This document describes how to install the `aptos` CLI tool. See [Use Aptos CLI](use-aptos-cli) for how to use the CLI.
+
+## Install precompiled binary (easy mode)
+
+* Navigate to the [release page](https://github.com/aptos-labs/aptos-core/releases) for Aptos CLI.
+* Download the latest release for your computer.
```
Comment (davidiw) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:14]: your platform?

**Review by davidiw — APPROVED** [review]
@gregnazario

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+* Place this at a location for you to run it e.g. `~/bin/aptos` in Linux.
+* On Linux and Mac, make this executable `chmod +x ~/bin/aptos`.
```
Comment (gregnazario) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:16]: We might have to note permissions from mac on allowing the executable outside of just this.

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+2. Install the `aptos` CLI tool by running the below command.  You can run this command from any directory.  The `aptos`
+   CLI tool will be installed into your `CARGO_HOME`, usually `~/.cargo`:
+```bash
+cargo install --git https://github.com/aptos-labs/aptos-core.git aptos --tag aptos-cli-0.2.0
+```
+3. Confirm that the `aptos` CLI tool is installed successfully by running the below command.  The terminal will display
+   the path to the `aptos` CLI's location.
+```bash
+which aptos
+```
```
Comment (gregnazario) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:56]: I do want at some point at the end of this to add how to install autocompletions.  But we can do that after this PR.

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md [hunk 5]
```diff
@@ -0,0 +1,732 @@
...
+This document describes how to use the `aptos` CLI tool. To install the CLI, see [Install Aptos CLI](install-aptos-cli).
+
+### Command Line Help
+Command line help is available.  Type `aptos help` or `aptos --help` to see the available command options.
+```bash
+$ aptos help
+aptos 0.2.0
+Aptos Labs <opensource@aptoslabs.com>
+CLI tool for interacting with the Aptos blockchain and nodes
+
```
Comment (gregnazario) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:18]: This looks verbatim.  I want to spend some time updating the docs after this is pushed to reflect some updates

**Review by gregnazario — APPROVED** [review]

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md (hunk 3 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:16]: Added this to this Linear issue to address this in a subsequent PR: https://linear.app/aptoslabs/issue/DOC-45/update-aptos-cli-doc

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md (hunk 4 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:56]: Yes. This PR, as we talked, is simply moving the README doc into this site, without touching the doc. I have another ticket for updating this doc: https://linear.app/aptoslabs/issue/DOC-45/update-aptos-cli-doc

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md (hunk 5 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md:18]: Yes. https://linear.app/aptoslabs/issue/DOC-45/update-aptos-cli-doc is our reference ticket.

**Pull request merged**